src/sc_kpm/sc_keynodes.py

```diff
@@ -56,6 +57,12 @@ def resolve(cls, identifier: Idtf, sc_type: Optional[ScType]) -> ScAddr:
             cls._logger.debug("Resolved %s identifier with type %s: %s", repr(identifier), repr(sc_type), repr(addr))
         return addr
 
+    def rrel(cls, index: int) -> ScAddr:
+        """Get rrel_i node. Max rrel index is 10"""
+        if index > cls._max_rrel_index:
+            raise KeyError(f"You cannot use rrel more than {cls._max_rrel_index}")
+        return cls.resolve(f"rrel_{index}", NODE_CONST_ROLE)  # pylint: disable=no-value-for-parameter
+
```

Comment:
role relation can be used not only with indexes. So if you want to use it only for indexes -- rename this, maybe to the _rrel_index()_ or smth like that. Also add check for input value, right now I can pass a string here

Reply:
Done